core/src/main/resources/filodb-defaults.conf

```diff
@@ -183,6 +183,33 @@ filodb {
       downsample-schema = "delta-histogram"
     }
 
+    prom-histogram-min-max {
```

Comment:
Do you want to call it prom anymore or something like otel? Technically Prometheus histograms dont have min max and prom stands for Prometheus.

Reply:
I like the idea of calling it `otel_histogram`.

Reply:
updated to Otel-histogram